db/types.go

```diff
@@ -73,8 +74,9 @@ func (n *NftEvent) Attach(payload EventPayload) {
 
 type PageRequest struct {
 	Key     uint64 `form:"key"`
-	Limit   uint64 `form:"limit,default=100" binding:"gte=1,lte=100"`
+	Limit   int    `form:"limit,default=100" binding:"gte=1,lte=100"`
```

Comment:
should this be a uint?

Reply:
Use int because it is the most common type in Go (ex: len())